app/views/elevator_mailer/notification.text.erb

```diff
@@ -1,9 +1,20 @@
 <%= I18n.t("email.elevator.pending_notification_subject") %>
 ========================================
 
-<%= I18n.t("email.elevator.start") %>: <%= @elevator_booking.start %>
-<%= I18n.t("email.elevator.end") %>: <%= @elevator_booking.end %>
+<% if @elevator_booking.moveType == 2 %>
+  <%= I18n.t("email.elevator.move") %>
+<% else %>
+  <%= I18n.t("email.elevator.delivery") %>
+<% end %>
+<%= I18n.t("email.elevator.start") %>: <%= @elevator_booking.start.in_time_zone("Eastern Time (US & Canada)").strftime("%B %-d, %Y %l:%M %p") %>
```

Comment:
May want to also time format for approval emails

Reply:
Oh! Yes!! Good eye. Thanks!!